libobs/obs.h

```diff
@@ -2553,6 +2553,12 @@ EXPORT void obs_encoder_set_audio(obs_encoder_t *encoder, audio_t *audio);
  */
 EXPORT video_t *obs_encoder_video(const obs_encoder_t *encoder);
 
+/**
+ * Returns the parent video output context used with this encoder, or NULL if not
+ * a video context. Used when FPS divisor is enabled
```

Comment:
```suggestion
 * a video context. Used when FPS divisor is enabled.
```

Though, I confess, I'm not sure I understand the second sentence. I only see the function defined, but not used anywhere.

Reply:
Added further context to the description. I think it should make more sense now.